parcel.go

```diff
@@ -36,25 +59,56 @@ func (s ParcelStore) GetByClient(client int) ([]Parcel, error) {
 	// заполните срез Parcel данными из таблицы
 	var res []Parcel
 
+	rows, err := s.db.Query(
+		"SELECT number, client, status, address, created_at FROM parcel WHERE client = :client",
+		sql.Named("client", client))
+	if err != nil {
+		return res, err
```

Comment:
Если ошибка, лучше вернуть `nil`, так как данные могут быть битыми
```suggestion
		return nil, err
```